### PR TITLE
Install requests for occasional tasks

### PR DESCRIPTION
**Pull request by joshmoore**
Useful for writing tests and scripting usage of the API.

### CODE REVIEW

**Comment by joshmoore** [comment]
cc: @manics

File: .travis.yml [hunk 1]
```diff
@@ -1,29 +1,15 @@
...
 install:
-  # Install ansible
-  - pip install ansible
-
-  # Check ansible version
-  - ansible --version
-
-  # Create ansible.cfg with correct roles_path
-  - printf '[defaults]\nroles_path=../' >ansible.cfg
+  - pip install ansible docker-py molecule
```
Comment (manics) [inline .travis.yml:9]: FYI I've been capping molecule to 1.20 (following a breaking change from 1.19), but if you're using mostly default config it shouldn't matter.

File: tests/test_default.py [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import testinfra.utils.ansible_runner
+
+testinfra_hosts = testinfra.utils.ansible_runner.AnsibleRunner(
+    '.molecule/ansible_inventory').get_hosts('all')
+
+
+def test_hosts_file(File):
```
Comment (manics) [inline tests/test_default.py:7]: This is the default example molecule test. Does molecule still run successfully if there aren't any test files?

**Review by manics — COMMENTED** [review]
Originally this was just python deps for the OMERO.server. I'm happy for this to be extended to deps for OMERO related scripts (though I think anything that's only for testing should be separate since it more dev than production). Do you want to update the readme to make this clearer?

**Comment by joshmoore** [comment]
> Originally this was just python deps for the OMERO.server.

Perhaps better in the web role, then? cc: @jburel @aleksandra-tarkowska

File: tests/test_default.py (hunk 2 above)
Comment (joshmoore) [reply tests/test_default.py:7]:
```
================= 2 passed, 1 pytest-warnings in 3.25 seconds ==================
```

vs.

```
Idempotence test passed.
```

so I think so.
```

File: .travis.yml (hunk 1 above)
Comment (joshmoore) [reply .travis.yml:9]: Reminds me that we probably need a list of the capped versions that we periodically review.

**Comment by jburel** [comment]
web-role might be more suitable in that case

**Comment by joshmoore** [comment]
Ok. Closing in favor of just activating molecule here. Will push everything to web then.

**Pull request closed without merging**